src/app/services/order.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from "@angular/core";
+import { Observable } from "rxjs/internal/Observable";
+import { AngularFireDatabase } from "@angular/fire/database";
+import { Order } from "../models/order";
+
+@Injectable({
+  providedIn: "root"
+})
+export class OrderService {
+  items: Observable<any[]>;
+  orders: any;
+  private db: any;
+
+  constructor(db: AngularFireDatabase) {
+    this.db = db;
+  }
+
+  getOrder(): Observable<Order[]> {
```

Comment:
jest już podobna metoda w innym serwisie

Reply:
getItem w product service wystarczy dodać jeszcze dwa parametry za this.items i za this.products

Reply:
Zapomniałem wywalić ten serwis jak później zrobiłem firebase.service